src/main/webapp/resources/js/pages/sequence-files/index.js

```diff
@@ -18,24 +17,26 @@ See: https://webpack.js.org/guides/public-path/#on-the-fly
  */
 __webpack_public_path__ = setBaseUrl(`/dist/`);
 
+function App() {
+  const [path, route] = getRootPath(location.pathname);
+
+  return (
+    <Routes>
+      <Route path={path} element={<FastQC current={route} />}>
+        <Route path="overrepresented" element={<OverRepresentedSequences />} />
+        <Route path="details" element={<FastQCDetails />} />
+        <Route path="charts" element={<FastQCCharts />} />
```

Comment:
Lets combine these two (`<Route path="charts" element={<FastQCCharts />} />` and `<Route path="summary" element={<FastQCCharts />} />`) into `<Route path="*" element={<FastQCCharts />} />`

Reply:
Cannot do that for the reason I told you yesterday where the `*` does not capture the `/` router

Reply:
Whoops my bad. Thanks

Reply:
566b889ac70a7df4bc8cd8ea74c173441ac8cab6